app/models/manageiq/providers/openstack/inventory/collector.rb

```diff
@@ -100,4 +100,21 @@ def volume_service
   def orchestration_service
     @orchestration_service ||= manager.openstack_handle.detect_orchestration_service
   end
+
+  def all_orchestration_stacks
+    return [] unless orchestration_service
+    # TODO(lsmola) We need a support of GET /{tenant_id}/stacks/detail in FOG, it was implemented here
+    # https://review.openstack.org/#/c/35034/, but never documented in API reference, so right now we
+    # can't get list of detailed stacks in one API call.
```

Comment:
I tested Fog and also Heat API, and OpenStack still doesn't support list stacks with details. Did I missed something or we need add get_stack with details iterated queries?


https://github.com/openstack/heat/blob/master/heat/api/openstack/v1/stacks.py

Reply:
ah, this TODO is unrelated. It's just copied when I moved the method. (it was not implemented for 5 years or so? :-))

Reply:
you can move this method back to `app/models/manageiq/providers/openstack/inventory/collector/cloud_manager.rb` once it is not used in targeted refresh

Reply:
Right, it has not been implemented since :-)

Okay, thanks.